src/test/omap_bench.cc

```diff
@@ -84,9 +84,9 @@ int OmapBench::setup(int argc, const char** argv) {
       	   << "rand for random sizes between\n"
       	   << "                        0 and max size, uniform for all sizes"
       	   << " to be specified size.\n"
-      	   << "                        (default "<<value_size;
-      cout <<"\n  --name          the rados id to use (default "<<rados_id;
-      cout<<")\n";
+           << "                        (default uniform)\n";
+      cout << "	--name          the rados id to use (default "<<rados_id
```

Comment:
Nope, it's a tab, if you can do "set list" in vim, you can see that other lines are all tabs, that why the "--name" option is not aligned with others. 
While the space before rados_id, anyway, I didn't add the space just want to keep consistent with others above it, like "threads", "objests", etc.